coordinator/internal/logic/provertask/prover_task.go

```diff
@@ -63,6 +72,14 @@ func (b *BaseProverTask) checkParameter(ctx *gin.Context, getTaskParameter *coor
 		return nil, fmt.Errorf("incompatible prover version. please upgrade your prover, minimum allowed version: %s, actual version: %s", b.cfg.ProverManager.MinProverVersion, proverVersion.(string))
 	}
 
+	// if the prover than v4.3.41-144c7ed-8f17df8-6e1a5cd and param forkBlockNumber is empty, represents
```

Comment:
Why this version?

Reply:
This change is not really related to hard forks, right? If not, would it make sense to move it to its own PR?

Reply:
ok, just a additional check, not very important, can remove it.

Reply:
fixed [49ae3c0](https://github.com/scroll-tech/scroll/pull/1182/commits/49ae3c0ceed33ce259d1adeef9980f912ae5cdfe)